src/components/Calculator.jsx

```
@@ -0,0 +1,107 @@
+                        setDisplay('');
+                        setTimeout(() => setError(''), 2000);
+                    }
+                } else {
+                    // No user logged in, use default code or just unlock
+                    if (enteredCode === '1234=') {
+                        onUnlock(enteredCode);
+                        setDisplay('');
+                        setError('');
+                    } else {
+                        // Try to evaluate as math
+                        setDisplay(eval(display).toString());
```

Biome (2.3.14)

[error] eval() exposes to security risks and performance issues.

See the MDN web docs for more details.
Refactor the code so that it doesn't need to call eval().

(lint/security/noGlobalEval)